src/dsymbol.d

```diff
@@ -1207,6 +1207,9 @@ private:
     Dsymbols* importedScopes;
     PROTKIND* prots;            // array of PROTKIND, one for each import
 
+    import std.bitmanip : BitArray;
+    BitArray accessiblePackages;// whitelist of accessible (imported) packages
```

Comment:
I remember that being said, but certainly not that it was the conclusion.  Switching to D gives us plenty of productivity improvements without using phobos.

Currently dmd compilation is completely independent of changes in phobos, and this doesn't look like a compelling enough case to be worth breaking that.

> As if the layout of BitArray would ever need to change.

It doesn't matter.  There is no way we should be relying on the internal layout of a phobos type.  If you really can't live without it, you might as well copy it into root, since you're effectively adding the implementation to dmd's source anyway.